addons/base_iban/static/src/tests/iban_widget.test.js

```diff
@@ -0,0 +1,74 @@
+import { ResPartnerBank } from "./mock_server/mock_models/res_partner_bank";
+
+import { DELAY } from "@base_iban/components/iban_widget/iban_widget";
+import { test } from "@odoo/hoot";
+import { advanceTime } from "@odoo/hoot-mock";
+import {
+    click,
+    contains,
+    defineMailModels,
+    insertText,
+    openFormView,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+import { defineModels, onRpc } from "@web/../tests/web_test_helpers";
+
+defineModels([ResPartnerBank]);
+defineMailModels();
```

Comment:
We should not promote this pattern as it does not handle well duplicates.
```suggestion
defineModels([...mailModels, ResPartnerBank]);
```

Reply:
Also because calling `defineModels` multiple times can lead to issues in model definitions